packages/core/src/util/resolvers.ts

```diff
@@ -134,13 +134,29 @@ export const resolveSchema = (
       const schemas = [].concat(
         resultSchema?.oneOf ?? [],
         resultSchema?.allOf ?? [],
-        resultSchema?.anyOf ?? []
+        resultSchema?.anyOf ?? [],
+        // also add root level schema composition entries
+        schema?.oneOf ?? [],
+        schema?.allOf ?? [],
+        schema?.anyOf ?? []
```

Comment:
Can you explain the use cases where this is necessary?

Reply:
I'll try to explain this in englisch. I hope it is comprehensible.

The `resultSchema` variable already holds the first resolved path entry. Which mostyl is `properties`.
If we take for example, the example given in the corresponding test-file `resolvers.test.ts` there I defined an `anyOf` in the root of the schema. The problem was that this code would now search further inside `properties` for the scopes defined inside `anyOf` - where it would not find those. That is why we need to also add the root level compositions here.